php_sql/task1/index.php

```diff
@@ -0,0 +1,73 @@
+<?php 
+    $valid=array();
+    $error=array();
+    $maxSizeCourse =150;
+    $maxSizeName=200;
+    $minLengthDescription=10;
```

Comment:
Nice job to extract these values in separate variables, but in this task the maximum length for the title and teacher fields is 128, and for the description field is 1024.